plugins/talk-plugin-offtopic/client/containers/OffTopicTag.js

```diff
@@ -0,0 +1,15 @@
+import {gql} from 'react-apollo';
+import OffTopicTag from '../components/OffTopicTag';
+import {withFragments} from 'plugin-api/beta/client/hocs';
+
+export default withFragments({
+  comment: gql`
+    fragment TalkOfftopic_OffTopicTag_comment on Comment {
```

Comment:
I don't foresee that this query will cause any issues. Multiple similar fragments might be added to the query, but graphql can handle this, and the fragment is needed to filter, optimise the component, and decouple the component.